docs/learn-about-neptune/collaborate.rst

```diff
@@ -0,0 +1,238 @@
+Collaborating in Neptune
+========================
+
+As an experiment tracking hub, Neptune provides several features for enabling knowledge sharing and collaboration among members
+of your data science team.
+
+To share a view on a project or any of its parts, you simply copy and paste the URL to it.
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: top
+
+About Access to Organizations and Projects
+------------------------------------------
+
+- If the project is private, access to the shared content is restricted to users who have access to the project.
+- If the project is public, anyone who has access to the Internet has access to the shared content.
+
+For more information about user permissions in organizations and projects, see :ref:`user roles <core-concepts_user-roles>`.
+
+Sharing View Links with Teammates
+---------------------------------
+
+Neptune lets you share views of experiments, Notebooks, and projects by sharing the URL to a specific view.
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: top
+
+
+To get the URL you need for sharing, copy the URL from the address bar.
+
+The URL changes in accordance with the particular view you are in. Details are provided below.
+
+Experiment details
+^^^^^^^^^^^^^^^^^^
+
+You can share short experiment IDs. An experiment ID is a unique identifier within the project.
+
+In the **Experiment Details** tab, under the **Metadata** section, click the **Copy** button in the ID field.
+
+**Example**
+
+https://ui.neptune.ai/o/USERNAME/org/example-project/e/HELLO-21/details
+
+Experiment charts and other resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can share experiment charts and other resources.
+
+Inside the experiment, there are tabs on the left side: Charts, Logs, Monitoring, Artifacts, Source code, Parameters, Details.
+Each displays specific content that is logged to Neptune for the specific experiment. Each tab has a unique URL.
```

Comment:
I was expecting that you'll insert the following here:
```
- Charts: https://ui.neptune.ai/USERNAME/example-project/e/HELLO-191/charts
- Logs: https://ui.neptune.ai/USERNAME/example-project/e/HELLO-191/logs
- Monitoring: https://ui.neptune.ai/USERNAME/example-project/e/HELLO-21/monitoring
- Artifacts: https://ui.neptune.ai/USERNAME/example-project/e/HELLO-191/artifacts
- Source code: https://ui.neptune.ai/USERNAME/example-project/e/HELLO-191/source-code
- Parameters: https://ui.neptune.ai/USERNAME/example-project/e/HELLO-191/parameters
- Details: https://ui.neptune.ai/USERNAME/example-project/e/HELLO-191/details
```

I'm not 100% convinced it's valuable, but it was there at the beginning, and now it's not. Up to you.

Reply:
I agree. Something is lost without it.
I'll put it back.